Assets/Scripts/SS3D/Systems/Inventory/Containers/Inventory.cs

```diff
@@ -54,6 +72,14 @@ public override void OnStartClient()
 
             InventoryView.Setup();
             InventoryView.Enable(true);
+
+            CmdPlayerContainersReady(Body);
+        }
+
+        [ServerRpc]
+        private void CmdPlayerContainersReady(Entity entity)
+        {
+            new PlayerContainersReady(entity).Invoke(this);
```

Comment:
Call the system method directly, its not correct to network an event bus

Reply:
No idea what you mean lol